src/io/github/sspanak/tt9/ime/TraditionalT9.java

```diff
@@ -271,6 +274,11 @@ protected boolean onRight(boolean repeat) {
 	protected boolean onNumber(int key, boolean hold, int repeat) {
 		String currentWord = getComposingText();
 
+		if(mInputMode.isABC()) {
+			delayHandler.removeCallbacks(delayAction);
+			delayHandler.postDelayed(delayAction, 800);
```

Comment:
Let's make this an on/off setting under Keyboard -> ABC Mode. Personally, I find it confusing when the keyboards auto-accept letters or words, so I would like to keep it turned off.

Reply:
Hello, I will have to apologize, I am quite new to GitHub and didn't realize these changes would get included in the pull request for the Swedish language.

My intention was to only use the changes for Traditional T9 and Hotkeys in my own branch. Possibly contributing with the delay function later on with on/off option and adjustable delay.

Reply:
But I do want to include this feature for other people to use it. It's nice that you shared it. :slightly_smiling_face:

Reply:
Thanks, that sounds wonderful if you have the time! :)